src/opts/cli.rs

```diff
@@ -78,60 +86,38 @@ pub fn command(scale_help: String, default_theme: Option<ThemeType>) -> Command
         .arg(theme_arg)
         .arg(scale_arg)
         .arg(gen_comp_arg)
+        .arg(config_arg)
         .arg(page_width_arg)
 }
 
 impl Args {
-    const SCALE_HELP: &str = "Factor to scale rendered file by";
-
-    pub fn new(config: &Config) -> Self {
+    pub fn new() -> Self {
         let program_args = std::env::args_os().collect();
-        Self::parse_from(program_args, config)
-    }
-
-    pub fn program_args(&self) -> Vec<String> {
-        let mut args = Vec::new();
-        args.push(self.file_path.as_os_str().to_str().unwrap().to_string());
-        if let Some(theme) = self.theme {
-            args.push("--theme".to_owned());
-            args.push(theme.as_str().to_owned());
-        }
-        if let Some(scale) = self.scale {
-            args.push("--scale".to_owned());
-            args.push(scale.to_string());
-        }
-        args
+        Self::parse_from(program_args)
     }
 
-    pub fn parse_from(args: Vec<OsString>, config: &Config) -> Self {
-        let scale_help = format!(
-            "{} [default: {}]",
-            Self::SCALE_HELP,
-            match config.scale {
-                Some(scale) => scale.to_string(),
-                None => String::from("Window's scale factor"),
-            }
-        );
-
-        let c = command(scale_help, config.theme);
+    pub fn parse_from(args: Vec<OsString>) -> Self {
+        let c = command();
         let matches = c.get_matches_from(args);
 
         // Shell completions exit early so handle them first
         if let Some(shell) = matches.get_one::<Shell>("shell").copied() {
-            let mut c = command(Self::SCALE_HELP.to_owned(), None);
+            let mut c = command();
             Self::print_completions(shell, &mut c);
             std::process::exit(0);
         }
 
         let file_path = matches.get_one("file").cloned().expect("required");
         let theme = matches.get_one("theme").cloned();
         let scale = matches.get_one("scale").cloned();
-        let page_width = matches.get_one("page_width").cloned();
+        let config = matches.get_one("config").cloned();
+        let page_width = matches.get_one("page width").cloned();
```

Comment:
```suggestion
        let page_width = matches.get_one("page_width").cloned();
```

Other part of the last comment